<launch>
    <node name="simulator" pkg="hockey_cup" type="simulator.py" output="screen"/>
    <node name="diff_drive" pkg="hockey_cup" type="diff_drive.py" output="screen"/>
</launch>